wiki/Beatmap/Marathon/de.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Marathon
+
+Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) mit einer Länge von mehr als fünf Minuten zu beschreiben. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
+
+## Geschichte
+
+Als Marathon-Beatmaps zuerst eingeführt wurden, mussten sie mindestens *sechs* Minuten lang sein und durften nur eine einzelne Schwierigkeitsstufe enthalten (mit der Ausnahme, dass jede Beatmap noch eine zusätzliche [osu!taiko](/wiki/Game_mode/osu!taiko)-Schwierigkeitsstufe beinhalten konnte).
+
+Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und nicht in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

Comment:
```suggestion
Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und zu verhindern, dass die Beatmaps in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet wurden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

Reply:
Is there a problem with `zu werden` here? I'd prefer something like this:

```suggestion
Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und um zu verhindern, in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

Reply:
Both of the suggestions above are fine to me.

There's something else I would like to mention.

```suggestion
Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Einreichungssystem zu umgehen und um zu verhindern, in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Schlussendlich wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
```

Firstly, I translated BSS in [Beatmapping/Beatmap submission](https://osu.ppy.sh/wiki/de/Beatmapping/Beatmap_submission) with `Beatmap-Einreichungssystem`.

Secondly, `Später` could be changed to `Schlussendlich` to have more word variety.